README.md

```diff
@@ -30,23 +30,91 @@ This repository implements techniques for doing a per-step fault tolerance so
 you can keep training if errors occur without interrupting the entire training
 job.
 
-This is based off of the large scale training techniques presented at PyTorch
-Conference 2024.
+[This is based on the large scale training techniques presented at PyTorch
+Conference 2024.](./media/fault_tolerance_poster.pdf)
+
+## Overview
+
+torchft is designed to provide the primitives required to implement fault
+tolerance in any application/train script as well as the primitives needed to
+implement custom fault tolerance strategies.
+
+Out of the box torchft provides the following algorithms:
+
+* Fault Tolerant DDP
+* Fault Tolerant HSDP: fault tolerance across the replicated dimension with any mix of FSDP/TP/PP/CP/EP across the other dimensions.
```

Comment:
For PTD PP + HSDP, I'm not sure if we already support it with TorchFT. We need some investigations about the gaps.

Reply:
changed to just `etc`